app/views/staff/grids/schedule.json.jbuilder

```diff
@@ -0,0 +1,10 @@
+
+json.rooms current_event.rooms, :id, :name
+json.days (1..current_event.days).to_a
+json.slots current_event.time_slots, :conference_day, :id, :update_path, :start_time, :end_time, :program_session_id, :track_id, :room_id
+json.sessions current_event.program_sessions
+json.unscheduledSessions current_event.program_sessions.unscheduled
+json.counts EventStats.new(current_event).schedule_counts
+json.tracks current_event.tracks.sort_by_name
+json.csrf session[:_csrf_token]
+json.bulkPath Rails.application.routes.url_helpers.event_staff_schedule_grid_bulk_time_slot_path(current_event)
```

Comment:
I am not sure but you may be already have access to all url_helpers here so you can try just making this line: 
```
json.bulkPath event_staff_schedule_grid_bulk_time_slot_path(current_event)
```

Reply:
Your hunch was correct! Refactored for the next push.